frontend/src/utils/StudentUtils.ts

```diff
@@ -47,25 +47,36 @@ export const initializeAnswers = (
 const getCompletedCount = (
   prevAnswer: Answers,
   answerElementIndex: number,
-  value: number[] | undefined,
+  value: (number | undefined)[] | undefined,
 ) => {
   let offset = 0;
   const prevCompletedCount = prevAnswer.completedCount;
   const isPrevCompletedElement: boolean =
-    prevAnswer.elements[answerElementIndex].elementAnswers.length !== 0;
+    prevAnswer.elements[answerElementIndex].elementAnswers.length !== 0 &&
+    !prevAnswer.elements[answerElementIndex].elementAnswers.includes(undefined);
```

Comment:
context: 

fraction answer looks like `[1, undefined]` for 
<img width="66" alt="image" src="https://github.com/uwblueprint/jump-math/assets/69697180/ab8ad595-a21d-4601-98c2-9163e8c76e9f">

fraction answer looks like `[undefined, 1, undefined]` for 
<img width="108" alt="image" src="https://github.com/uwblueprint/jump-math/assets/69697180/2630f3a8-0d1f-45be-9aca-30551f6505f7">